doc/pki.md

```diff
@@ -0,0 +1,28 @@
+# GOV.UK Verify UK Proxy Node PKI
+
+The eIDAS Proxy Node Public Key Infrastructure (PKI) is detailed in the diagram below.
+It addresses SAML encryption/decryption, signing/validation, and publishing of PKI metadata between the UK Proxy Node and an EU connecting country.
+The key describes a sequential flow of an eIDAS SAML message
+
+![](images/proxy_node_pki.svg)
+
+Key to Diagram:
+
+| Step      | Description |
+| :----:       |    :---     |
+| a      | At startup, the `metadata-controller` connects to `CloudHSM` to produce a signing keypair. It publishes the public key on `metadata`
```

Comment:
The metadata controller also generates a keypair for signing the generated metadata itself (as well as the signing key used by the translator) ... or at least it should do, I think it might be using the same keypair for both right now